lms/static/js/Markdown.Converter.js

```diff
@@ -1285,27 +1285,12 @@ else
         //  attacklab: Utility functions
         //
 
-        var _problemUrlChars = /(?:["'*()[\]:]|~D)/g;

```

Comment:
Actually, we could have alternatively updated the library which contains three files but one of them has changes at many places.